lib/nerves_ssh/options.ex

```diff
@@ -40,7 +41,8 @@ defmodule NervesSSH.Options do
           daemon_option_overrides: keyword()
         }
 
-  defstruct authorized_keys: [],
+  defstruct name: :default,
```

Comment:
I'm going back and forth on having the default name just continue to be `NervesSSH` rather than `:default` here. It seems like that would be much less of an interrupt than using a new atom of `:default` since that has been the expected name so far for the life of this app. What do you think?

Reply:
I think that's fine, done 👍